test/pool_test.dart

```diff
@@ -697,6 +697,11 @@ void main() {
       });
     });
   });
+
+  test("throw error when pool limit <= 0", () {
```

Comment:
Single quotes for consistency with surrounding code

Reply:
This is merged from a user commit. We should have a follow-up to just move to single quotes.

For now, not specific to this CL